ramls/packages.raml

```diff
@@ -66,7 +66,7 @@ types:
       Content-Type:
         example: application/vnd.api+json
     body:
-      application/vnd.api+json:
+      application/json:
```

Comment:
As discussed, let's please wait on this until we hear back from RMB folks! Thanks!

Reply:
Looks like Adam has merged related PR in RMB project

Reply:
updated the code and changed the body type to application/vnd.api+json: